<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplace</class>
 <widget class="QDialog" name="FindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>227</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="lyDialogLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="twTabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FindTab">
      <attribute name="title">
       <string>Fin&amp;d</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="wSearch" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leFind"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbFindWhat">
              <property name="text">
               <string>Fi&amp;nd what:</string>
              </property>
              <property name="buddy">
               <cstring>leFind</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QStackedWidget" name="swReplaceLabelHider">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="wShowPageField">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="lbReplaceWith">
                  <property name="text">
                   <string>Replace w&amp;ith:</string>
                  </property>
                  <property name="buddy">
                   <cstring>leReplace</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="wHidePageField"/>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QStackedWidget" name="swReplaceFieldHider">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="wShowPageLabel">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="leReplace"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="wHidePageLabel"/>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btMore">
              <property name="text">
               <string>&amp;More</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btReplace">
              <property name="text">
               <string>&amp;Replace</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btReplaceAll">
              <property name="text">
               <string>Replace &amp;All</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btCount">
              <property name="text">
               <string>&amp;Count</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btFindNext">
              <property name="text">
               <string>&amp;Find Next</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wOptions" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="cbWholeWord">
              <property name="text">
               <string>Match whole word onl&amp;y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbMatchCase">
              <property name="text">
               <string>Matc&amp;h case</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Search mode</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="rbNormalSearch">
               <property name="text">
                <string>Nor&amp;mal</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbRegexSearch">
               <property name="text">
                <string>Re&amp;gular expression</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Direction</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="rbUpDirection">
               <property name="text">
                <string>&amp;Up</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbDownDirection">
               <property name="text">
                <string>D&amp;own</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ReplaceTab">
      <attribute name="title">
       <string>Re&amp;place</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leFind</tabstop>
  <tabstop>leReplace</tabstop>
  <tabstop>btMore</tabstop>
  <tabstop>btReplace</tabstop>
  <tabstop>btReplaceAll</tabstop>
  <tabstop>btCount</tabstop>
  <tabstop>btFindNext</tabstop>
  <tabstop>cbWholeWord</tabstop>
  <tabstop>cbMatchCase</tabstop>
  <tabstop>rbNormalSearch</tabstop>
  <tabstop>rbRegexSearch</tabstop>
  <tabstop>rbUpDirection</tabstop>
  <tabstop>rbDownDirection</tabstop>
  <tabstop>twTabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
